templates/week.html

```diff
@@ -0,0 +1,26 @@
+{% extends "tabulator.html" %}
+{% block active_tab %}2{% endblock %}
+{% block tab_head %}
+<script src="../static/js/tab3.js" type="module"> </script>
+{% endblock %}
+{% block tab_content %}
+<!--If JS is not Enabled!-->
+<noscript>
+    <h3>
+        Woche
+    </h3>
+    <p class="pb-3 text-xl">F&uuml;r eine bessere Erfahrung, schalten Sie JavaScript ein und laden Sie die Seite neu.</p>
+    <img src="../api/img/temperature/week.jpg" alt="Temperatur Monat">
+    <img src="../api/img/rain/week.jpg" alt="Regen Monat">
+    <img src="../api/img/pressure/week.jpg" alt="Luftdruck Monat">
+    <img src="../api/img/wind_speed/week.jpg" alt="Windgeschwindigkeit Monat">
+</noscript>
+<div class="js_enabled" style="display: none;"  data-tab="2">
```

Comment:
Please only use inline style if needed and if so, please add a comment clarifying the requirement.